api/desecapi/views/tokens.py

```diff
@@ -88,7 +88,7 @@ def create(self, request, *args, **kwargs):
             raise Http404
 
     def get_queryset(self):
-        qs = Token.objects.filter(user=self.request.user)
+        qs = Token.objects.filter(owner=self.request.user)
```

Comment:
`user=request.user or owner=request.user`?

Reply:
At the time of this commit, which does not expose new functionality, it should be `owner`. Subsequent commits adjust this as functionality is changed.